TreeDB/background_vacuum_test.go

```
@@ -459,6 +459,182 @@ func TestBackgroundIndexVacuumNativeRootWriteForcesProbeAfterSkipCap(t *testing.
+	if err := d.Close(); err != nil {
+		t.Fatalf("close: %v", err)
+	}
+	if control.CommitInsert(epochID) || d.bgVac.Stats().DeferredVectorBuildActive {
+		t.Fatal("closed database admitted a deferred epoch")
+	}
+
+	reopened, err := Open(Options{Dir: dir, BackgroundIndexVacuumInterval: -1})
+	if err != nil {
+		t.Fatalf("reopen: %v", err)
+	}
+	defer reopened.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `reopened.Close` is not checked

(errcheck)